ethereum/core/src/test/java/tech/pegasys/pantheon/ethereum/core/TransactionTest.java

```diff
@@ -93,6 +93,11 @@ public void constantinople() {
     milestone("Constantinople");
   }
 
+  @Test
+  public void petersburg() {
+    milestone("Petersburg");
+  }
+
```

Comment:
Since the transaction tests haven't been updated for "Petersburg" we shouldn't add this.  I doubt there will be additional transaction tests added since they're just about parsing transactions, not about executing them and that's unaffected.

Reply:
Done